pom.xml

```diff
@@ -469,8 +469,13 @@
         <version.org.eclipse.angus.angus-activation>2.0.1</version.org.eclipse.angus.angus-activation>
         <version.org.eclipse.angus.angus-mail>2.0.2</version.org.eclipse.angus.angus-mail>
         <version.org.eclipse.jdt>3.32.0</version.org.eclipse.jdt>
+<<<<<<< Updated upstream
         <version.org.eclipse.microprofile>6.0</version.org.eclipse.microprofile>
         <version.org.eclipse.microprofile.config.api>3.1</version.org.eclipse.microprofile.config.api>
+=======
+        <version.org.eclipse.microprofile>6.1</version.org.eclipse.microprofile>
+        <version.org.eclipse.microprofile.config.api>3.0.3</version.org.eclipse.microprofile.config.api>
+>>>>>>> Stashed changes
```

Comment:
This looks like a merge gone wrong and this is what's breaking the build for this PR.